docs/faq.md

```diff
@@ -75,16 +75,8 @@ disabled-by-default counterpart W504. E203 should be disabled while changes are
 
 ## Does Black support Python 2?
 
-```{warning}
-Python 2 support has been deprecated since 21.10b0.
-
-This support will be dropped in the first stable release, expected for January 2022.
-See [The Black Code Style](the_black_code_style/index.rst) for details.
-```
-
-For formatting, yes! [Install](getting_started.md#installation) with the `python2` extra
-to format Python 2 files too! In terms of running _Black_ though, Python 3.6 or newer is
-required.
+Formatting Python 2 code support was removed in version 22.0. In terms of running
+_Black_, Python 3.6 or newer is required since the first release.
```

Comment:
```suggestion
_Black_, Python 3.6 or newer is required since the first non-beta release.
```
NIT: or "since 22.0"

Reply:
This sentence is all about _running_ black haha. Although perhaps this FAQ entry should be rewritten since it seems confusing to even co-maintainers :slightly_smiling_face:

Reply:
Oh I didn't think this through, has 3.6 been a runtime requirement really from the first ever Black release?

Reply:
Yup! Łukasz loves his f-strings far too much to support older versions :)

Reply:
```suggestion
_Black_, Python 3.6 or newer is required.
```
Perhaps we don't really need it.

Reply:
Fair, they are great 😄